src/coreclr/debug/ee/functioninfo.cpp

```diff
@@ -1518,6 +1518,9 @@ DebuggerJitInfo * DebuggerMethodInfo::FindJitInfo(MethodDesc * pMD,
     }
     CONTRACTL_END;
 
+#ifdef TARGET_ARM
+    addrNativeStartAddr = addrNativeStartAddr|THUMB_CODE;
```

Comment:
Can we use `DataPointerToThumbCode` or `PINSTRToPCODE` here?

Reply:
Sure, same logic, but I chose this one, in order to stay with same style as we have in related to those addresses code:
https://github.com/dotnet/runtime/blob/ecb4e6c178b98aec29fbf3e7a70a820240c1dd0b/src/coreclr/debug/ee/debugger.cpp#L2498-L2500
https://github.com/dotnet/runtime/blob/ecb4e6c178b98aec29fbf3e7a70a820240c1dd0b/src/coreclr/debug/daccess/dacdbiimpl.cpp#L1417-L1421